src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,131 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { v4 as uuidv4 } from 'uuid';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [peopleError, setPeopleError] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  const { personSlug } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => setPeopleError('Something went wrong'))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  const isError = !isLoading && peopleError;
+  const hasPeople = people.length > 0 && !isLoading;
+
+  const tableHeaders = ['Name', 'Sex', 'Born', 'Died', 'Mother', 'Father'];
+  const FEMALE_SEX = 'f';
+  const NO_PARENT_NAME = '-';
+
```

Comment:
Such variables should be named in `UPPER_CASE` and move them outside the component